libraries/c_sdk/standard/mqtt/src/iot_mqtt_serializer_deserializer_wrapper.c

```diff
@@ -248,8 +264,11 @@ IotMqttError_t _IotMqtt_connectSerializeWrapper( const IotMqttConnectInfo_t * pC
     if( status == IOT_MQTT_SUCCESS )
     {
         /* Allocating memory for Connect packet. */
-        networkBuffer.pBuffer = IotMqtt_MallocMessage( *pPacketSize );
-        networkBuffer.size = *pPacketSize;
+        if( pPacketSize != NULL )
```

Comment:
It is already asserted at the beginning of the function right? Why to repeat that step here?